Atomic/run.py

```diff
@@ -108,7 +108,9 @@ def run(self):
             if self.args.display:
                 return 0
             try:
-                db.pull_image(self.image, remote_image_obj)
+                err = db.pull_image(self.image, remote_image_obj)
+                if err:
+                    raise ValueError("Unable to pull image {}"..format(self.image))
```

Comment:
there is one extra '.' here

Reply:
doesn't it fail immediately once `pull_image` raises `CalledProcessError`?

Reply:
Nice catch, I will fix. And no, it doesn't seems to raise anything. I sent the PR first to see the tests running, but I didn't had time to test or validate it (as I reinstalled my system with enough disk).